content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -203,7 +203,7 @@ For a reference to old feature gates that are removed, please refer to
 | `TopologyManagerPolicyOptions` | `false` | Alpha | 1.26 | 1.27 |
 | `TopologyManagerPolicyOptions` | `true` | Beta | 1.28 | |
 | `UnknownVersionInteroperabilityProxy` | `false` | Alpha | 1.28 | |
-| `UserNamespacesStatelessPodsSupport` | `false` | Alpha | 1.25 | |
```

Comment:
This should not be removed.
It is something existed, please don't try to rewrite history.
You can instead write this:

```
| `UserNamespacesStatelessPodsSupport` | `false` | Alpha | 1.25 | 1.27 |
| `UserNamespacesStatelessPodsSupport` | `false` | Renamed | 1.28 | - |
```

And move the above two lines to the second table on this page.

Reply:
Oh, I didn't know we wanted to document old and not longer recognized feature gates. Did that now, thanks